pkg/util/gitlab/helper.go

```diff
@@ -0,0 +1,106 @@
+package gitlab
+
+type MergeCommentEvent struct {
```

Comment:
Why need to define this?

Reply:
https://github.com/zhujian7/cyclone/blob/bb7128664e20de4e6e0aa307620f93145e82c3d1/pkg/util/gitlab/gitlab.go#L27-L30